pkg/apis/core/validation/shoot.go

```diff
@@ -590,9 +629,13 @@ func ValidateKubernetesVersionUpdate(new, old string, fldPath *field.Path) field
 	return allErrs
 }
 
-func validateNetworkingUpdate(newNetworking, oldNetworking core.Networking, fldPath *field.Path) field.ErrorList {
+func validateNetworkingUpdate(newNetworking, oldNetworking *core.Networking, fldPath *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
 
+	if newNetworking == nil || oldNetworking == nil {
```

Comment:
In that case the normal validation error for `networking shouldn't be nil` will be returned. 
Because we call `ValidateShoot` also on `ValidateShootUpdate`